### PR TITLE
fix task direct link to logs

### PR DESCRIPTION
**Pull request by darcatron**
think this is the only usage. tested locally. worked fine
@ssalinas

### CODE REVIEW

File: SingularityUI/app/components/tasks/Columns.jsx [hunk 1]
```diff
@@ -361,8 +361,7 @@ export const LogLinkAndJSON = logPath => (
     cellRender={(taskId, rowData) => (
       <div className="hidden-xs">
         <OverlayTrigger placement="top" id="view-log-overlay" overlay={logTooltip}>
-          <Link
-            to={`request/${taskId.requestId}/tail/${logPath}?taskIds=${taskId.id}`}
+          <Link to={`task/${taskId.id}/tail/${Utils.substituteTaskId(config.runningTaskLogPath, taskId.id)}`}
```
Comment (ssalinas) [inline SingularityUI/app/components/tasks/Columns.jsx]: why `config.runningTaskLogPath` instead of `logPath` here? `logPath` is supplied as an argument.

Comment (darcatron) [inline SingularityUI/app/components/tasks/Columns.jsx]: my mistake, good catch. updated

**Comment by ssalinas** [comment]
👍 LGTM

**Comment by darcatron** [comment]
moved the path into a util func to prevent this issue from happening again

**Pull request merged**